clients/client-java/src/main/java/com/datastrato/gravitino/client/GravitinoAdminClient.java

```diff
@@ -203,7 +210,7 @@ public User addUser(String metalake, String user)
 
     UserResponse resp =
         restClient.post(
-            String.format(API_METALAKES_USERS_PATH, metalake, ""),
+            String.format(API_METALAKES_USERS_PATH, metalake, BLANK_PLACE_HOLDER),
```

Comment:
I feel like there will additional "/" at the end of the request path, right?

Reply:
The url will be `api/metalakes/%s/users/`. It's an acceptable url. It can share the same url pattern with get/post/put/delete.